We do not have these projects packaged.
They are not needed to run tests.

--- shellingham-1.5.4/Pipfile.orig
+++ shellingham-1.5.4/Pipfile
@@ -3,13 +3,9 @@
 
 [dev-packages]
 invoke = '*'
-parver = '*'
 pytest = '*'
 pytest-mock = '*'
-setl = '*'
-towncrier = '*'
 tox = '*'
-tox-gh-actions = '*'
 
 [scripts]
 release = 'inv release'
